File: barcode-creator-core-console-vb/PostalBarcode.vb
Imports System.IO
Imports ceTe.DynamicBarcode.Creator

Public Class PostalBarcode
    Public Shared Sub AustraliaPostBarcode(ByVal outputPath As String)
        Dim MyBarCode As AustraliaPost = New AustraliaPost("1139549554")
        MyBarCode.BackColor = New ceTe.DynamicBarcode.Creator.Rgb24(0, 191, 255)
        Dim MyBarCodeData As Byte() = MyBarCode.Draw(300, ImageFormat.Png)
        File.WriteAllBytes(outputPath, MyBarCodeData)
    End Sub

    Public Shared Sub DeutschePostIdent(ByVal outputPath As String)
        Dim MyBarCode As DeutschePostIdentcode = New DeutschePostIdentcode("12345678901", 100)
        MyBarCode.BarcodeMargin = New Margin(2)
        MyBarCode.Draw(outputPath, 300, ImageFormat.Png)
    End Sub

    Public Shared Sub DeutschePostLeit(ByVal outputPath As String)
        Dim MyBarCode As DeutschePostLeitcode = New DeutschePostLeitcode("1234567890123", 100)
        MyBarCode.BarColor = New ceTe.DynamicBarcode.Creator.Rgba32(170, 80, 0, 3)
        MyBarCode.Draw(outputPath, 300, ImageFormat.Png)
    End Sub

    Public Shared Sub IntelligentMail(ByVal outputPath As String)
        Dim MyBarCode As IntelligentMailBarcode = New IntelligentMailBarcode("0123456709498765432101234567891")
        MyBarCode.ShowText = True
        MyBarCode.FontSize = 2
        MyBarCode.Draw(outputPath, 300, ImageFormat.Png)
    End Sub

    Public Shared Sub KixBarcode(ByVal outputPath As String)
        Dim MyBarCode As Kix = New Kix("1231FZ13XHS")
        MyBarCode.SymbolHeight = 12
        MyBarCode.Draw(outputPath, 300, ImageFormat.Png)
    End Sub

    Public Shared Sub PostnetBarcode(ByVal outputPath As String)
        Dim MyBarCode As Postnet = New Postnet("20815470412")
        MyBarCode.Draw(outputPath, 300, ImageFormat.Png)
    End Sub

    Public Shared Sub Rm4sccBarcode(ByVal outputPath As String)
        Dim MyBarCode As Rm4scc = New Rm4scc("20815470412")
        MyBarCode.ShowText = True
        MyBarCode.TextAlign = Align.Right
        MyBarCode.Draw(outputPath, 300, ImageFormat.Png)
    End Sub

    Public Shared Sub SingaporePostBarcode(ByVal outputPath As String)
        Dim MyBarCode As SingaporePost = New SingaporePost("208154")
        MyBarCode.Draw(outputPath, 300, ImageFormat.Png)
    End Sub
End Class

File: barcode-creator-core-console-cs/PostalBarcode.cs
using ceTe.DynamicBarcode.Creator;
using System;
using System.IO;

namespace barcode_creator_core_console_cs
{
    class PostalBarcode
    {
        public static void AustraliaPostBarcode(string outputPath)
        {
            AustraliaPost barcode = new AustraliaPost("1139549554");
            barcode.BackColor = new ceTe.DynamicBarcode.Creator.Rgb24(0, 191, 255);
            Byte[] barcodeData = barcode.Draw(300, ImageFormat.Png);
            File.WriteAllBytes(outputPath, barcodeData);
        }
        public static void DeutschePostIdent(string outputPath)
        {
            DeutschePostIdentcode barcode = new DeutschePostIdentcode("12345678901", 100);
            barcode.BarcodeMargin = new Margin(2);
            barcode.Draw(outputPath, 300, ImageFormat.Png);
        }
        public static void DeutschePostLeit(string outputPath)
        {
            DeutschePostLeitcode barcode = new DeutschePostLeitcode("1234567890123", 100);
            barcode.BarColor = new ceTe.DynamicBarcode.Creator.Rgba32(170, 80, 0, 3);
            barcode.Draw(outputPath, 300, ImageFormat.Png);
        }
        public static void IntelligentMail(string outputPath)
        {
            IntelligentMailBarcode barcode = new IntelligentMailBarcode("0123456709498765432101234567891");
            barcode.ShowText = true;
            barcode.FontSize = 2;
            barcode.Draw(outputPath, 300, ImageFormat.Png);
        }
        public static void KixBarcode(string outputPath)
        {
            Kix barcode = new Kix("1231FZ13XHS");
            barcode.SymbolHeight = 12;
            barcode.Draw(outputPath, 300, ImageFormat.Png);
        }
        public static void PostnetBarcode(string outputPath)
        {
            Postnet barcode = new Postnet("20815470412");
            barcode.Draw(outputPath, 300, ImageFormat.Png);
        }
        public static void Rm4sccBarcode(string outputPath)
        {
            Rm4scc barcode = new Rm4scc("20815470412");
            barcode.ShowText = true;
            barcode.TextAlign = Align.Right;
            barcode.Draw(outputPath, 300, ImageFormat.Png);
        }
        public static void SingaporePostBarcode(string outputPath)
        {
            SingaporePost barcode = new SingaporePost("208154");
            barcode.Draw(outputPath, 300, ImageFormat.Png);
        }
    }
}
